README.md

```diff
@@ -80,8 +83,9 @@ Note: the `synthesis list` is a `json` file:
 containing a list of items with a) the path (relative to `in_dir`) of the source `wav` files;
 b) the target speaker (see `datasets/2019/english/speakers.json` for a list of options);
 and c) the target file name.
+Example usage:
 ```
-e.g. python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
+python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
 ```
 Voice conversion samples will be available soon.
```

Comment:
Might as well add another link to the samples too? It's nice to have the links near to the different sections as well.
```suggestion
Voice conversion samples are available [here](https://bshall.github.io/VectorQuantizedCPC/)
```

Reply:
I've made the changes below (I didn't realise you could commit it directly!).